sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -484,6 +500,7 @@ def expand(self, pcolls):
       pcoll_b = p | 'b' >> beam.Create(['b'])
       assert_that((pcoll_a, pcoll_b) | First(), equal_to(['a']))
 
+  @retry(2)
```

Comment:
I think you meant to decorate test_progress_metrics method instead.

Reply:
Oops. yes.